cartoframes/data/registry/base_dataset.py

```diff
@@ -86,7 +86,7 @@ def dataset_info(self):
 
         return self._dataset_info
 
-    def update_dataset_info(self, privacy=None, name=None):
+    def update_dataset_info(self, privacy=None, name=None, table_name=None):
```

Comment:
I can't see where `table_name` is being used and for.

Reply:
It is used in QueryDataset, but I need to add it in the parent and children too